_common/micro/healthChecks/postNodeStats.js

```diff
@@ -44,6 +45,11 @@ function _checkInputParams(bag, next) {
   var consoleErrors = [];
   bag.adapter = new ShippableAdapter('');
 
+  bag.defaultShell = '/bin/bash -c';
+
+  if (global.config.shippableNodeOperatingSystem === 'WindowsServer_2016')
```

Comment:
Since this is the second occurrence of this shell switching, let's move it to a global config setting that can be applied everywhere we need it.